src/libs/tools/global_toolbox.c

```diff
@@ -154,22 +236,6 @@ static void _lib_filter_grouping_button_clicked(GtkWidget *widget, gpointer user
 #endif // USE_LUA
 }
 
-static void _lib_overlays_button_clicked(GtkWidget *widget, gpointer user_data)
-{
-  gboolean show = gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(widget));
-  if(show)
-    gtk_widget_set_tooltip_text(widget, _("hide image overlays"));
-  else
-    gtk_widget_set_tooltip_text(widget, _("show image overlays"));
-  dt_conf_set_bool("lighttable/ui/expose_statuses", show);
-  dt_thumbtable_set_overlays(dt_ui_thumbtable(darktable.gui->ui), show);
-
-#ifdef USE_LUA
```

Comment:
I suppose we want this in  _overlays_toggle_button() and passing the boolean FALSE for DT_THUMBNAIL_OVERLAYS_NONE and TRUE otherwise.

It would be a pity to loose this. Later we can add mode information by passing the overlay mode in another lua call or updating the API but at least let's try to keep this.

Reply:
well in fact I've done TRUE if DT_THUMBNAIL_OVERLAYS_ALWAYS_NORMAL or DT_THUMBNAIL_OVERLAYS_ALWAYS_EXTENDED. and FALSE otherwise. So it'll mimic the old behavior little better.